apps/masterbots.ai/lib/helpers/ai-streams.ts

```
@@ -0,0 +1,97 @@
+  const jsonRegex = /data:\s*\{(?:[^{}]|(?:\{(?:[^{}]|(?:\{[^{}]*\}))*\}))*\}/g
+
+  try {
+    while (true) {
+      const { done, value } = await reader.read()
+      if (done) break
+
+      buffer += decoder.decode(value, { stream: true })
+
+      // console.log('Buffer: [Before Changes] => ', buffer)
+
+      let match
```

Biome

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)